### DISCUSSION TITLE
Reanimated animation in webpack app not working when used from a component library

### DISCUSSION THREAD

**Discussion by oliviercperrier** [software-mansion/react-native-reanimated#5477] [Q&A]
I have a react app using webpack and i have an external component library that uses react-native-reanimated.

When i use the library Spinner component (which use react-native-reanimated for the animatiion), the animation is not working.

When i copy the Spinner component to my react app, the animation is working. 

Not sure what im doing wrong or what is missing.

Thanks

**Comment by oliviercperrier** [comment]
Up

**Comment by MatiPl01** [comment]
Hey!
It's hard to tell what is wrong with your configuration without any repro. Would you be able to provide a minimum reproducible example or can you share more details about your configuration (like webpack config, versions of libraries used, etc.)?
